discovery-provider/src/monitors/monitors.py

```diff
@@ -59,6 +61,19 @@
     monitor_names.type: 'json'
 }
 
+DATABASE_INDEX_COUNT = {
+    monitor_names.name: monitor_names.database_index_count,
+    monitor_names.func: get_database_index_count,
+    monitor_names.type: 'int'
+}
+
+DATABASE_INDEX_INFO = {
+    monitor_names.name: monitor_names.database_index_info,
+    monitor_names.func: get_database_index_info,
+    monitor_names.type: 'json'
+}
```

Comment:
maybe let's add a custom TTL here for like 10 minutes? we shouldn't run this every minute

`monitor_names.ttl: 60 * 10`

?

or maybe even an hour or day...

Reply:
you right. made it six hours